README.md

```diff
@@ -0,0 +1,78 @@
+# Hydra
+
```

Comment:
Rename this file to something like `DOC.md` or whatever so that this doesn't get automatically rendered on the github repo's main page.